### PR TITLE
feat: change window to self for webworker support

### PR DESCRIPTION
**Pull request by dryajov**
Full WebWorker compatibility ipfs/js-ipfs#725

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "nodeify": "^1.0.0"
   },
   "devDependencies": {
-    "aegir": "^9.4.0",
+    "aegir": "^10.0.0",
```
Comment (dignifiedquire) [inline package.json]: Can you remove the `--webworker` flags in the scripts please, as they are not needed anymore

**Comment by dryajov** [comment]
@diasdavid @dignifiedquire can we get this merged and released? This is probably the dependency that will make most others fail once aegir is buped to v10.

**Comment by dignifiedquire** [comment]
Yes, lets ship this, was just waiting for CI to be happy :)

**Comment by dryajov** [comment]
awesome! 🎈 Is this released also?

**Comment by dignifiedquire** [comment]
yes, just shipped 0.4.1

**Pull request merged**